client/scss/components/_figure.scss

```diff
@@ -1,14 +1,18 @@
 //* @define figure
 
 .figure {
-  @include bleed(map-get($gutter-width, 'small'));
+  margin: 0;
 
-  @include respond-to('medium') {
-    @include bleed(map-get($gutter-width, 'medium'));
-  }
+  &:not(.figure--full) {
```

Comment:
This is quite a lot to have in a not - are we sure we want it on all other images, or should we create a new modifier?

Also - do we have an example of not `figure--full`?